pandas/hashtable.pyx

```diff
@@ -740,7 +719,7 @@ cdef class PyObjectHashTable(HashTable):
             int ret = 0
             object val
             khiter_t k
-            ndarray[int64_t] locs = np.empty(n, dtype=np.int64)
+            int64_t[:] locs = np.empty(n, dtype=np.int64)
```

Comment:
Was this change necessary or just for consistency with the rest of the HashTable implementations?